docs/semgrep-secrets/historical-scanning.md

```diff
@@ -0,0 +1,114 @@
+---
+slug: historical-scanning
+append_help_link: true
+title: Scan your git history
+hide_title: true
+description: Detect valid, leaked secrets in previous git commits through a historical scan.
+tags:
+  - Semgrep Secrets
+  - Team & Enterprise tier
+---
+
+# Scan your git history
+
+Detect valid, leaked secrets in previous git commits through a **historical scan**.
+
+You can perform one-time historical scans or enable historical scanning for all Secrets scans. Detecting valid secrets in your git history is a first step to reducing your repository's attack surface.
```

Comment:
Most orgs will probably start with running against the current version of their codebase, so I'm not sure if calling this a "first step" makes sense